index.html

```diff
@@ -0,0 +1,362 @@
+                Welcome to the Global Cryptocurrency Conference 2023, the premier gathering of innovators, experts, and enthusiasts, charting the future of digital currencies and blockchain technology.
+              </div>
+            </div>
+          </div>
+
+          <h3 class="subtitle is-3 mt-2">2023.10.15(THU) ~ 16(FRI) <br /></h3>
+          <p id="location">Las Vegas, Nevada</p>
+        </section>
+      </div>
+      <div class="program pb-6">
+        <div class="columns is-centered">
+          <div class="column is-narrow has-text-centered">
+            <h4
+              class="title is-4 has-text-white mt-2 mb-1 has-text-centered"
+              id="mainpg"
+            >
+              Main Program
+            </h4>
+          </div>
+        </div>
+        <div class="columns is-centered has-text-centered">
+          <div class="column mr-1 ml-1 mt-1 is-narrow mg-box is-2">
+            <span class="icon mt-2">
+              <i class="fas fa-cogs has-text-white-bis"></i>
+            </span>
+
+            <h4 class="title is-4 mt-1">DeFi</h4>
+
+            <div class="has-text-white-ter">
+              Decentralized Finance (DeFi): Innovations, Opportunities, and Challenges.
+            </div>
+          </div>
+
+          <div class="column mr-1 ml-1 mt-1 is-narrow mg-box is-2">
+            <span class="icon mt-2">
+              <i class="fas fa-coins has-text-white-bis"></i>
+            </span>
+            <h4 class="title is-4 mt-1">NFTs</h4>
+            <div class="has-text-white-ter">
+              The Evolution of Non-Fungible Tokens (NFTs): Beyond Digital Art and into the Mainstream.
+            </div>
+          </div>
+
+          <div class="column mr-1 ml-1 mt-1 is-narrow mg-box is-2">
+            <span class="icon mt-2">
+              <i class="fas fa-lock has-text-white-bis"></i>
+            </span>
+            <h4 class="title is-4 mt-1">Regulation</h4>
+            <div class="has-text-white-ter">
+              Regulatory Landscape for Cryptocurrencies: Balancing Growth and Security.
+            </div>
+          </div>
+
+          <div class="column mr-1 ml-1 mt-1 is-narrow mg-box is-2">
+            <span class="icon mt-2">
+              <i class="fas fa-leaf has-text-white-bis"></i>
+            </span>
+            <h4 class="title is-4 mt-1">Environmental Awareness</h4>
+            <div class="has-text-white-ter">
+              The Environmental Impact of Cryptomining: Sustainable Solutions for the Future
+            </div>
+          </div>
+        </div>
+        <div class="join is-hidden-desktop pt-4">
+        <button> Join CCG Conference</button>
+      </div>
+        <div class="columns is-centered">
+          <div class="column has-text-centered has-text-white-ter mt-3 mb-3">
+            <a href="" class="has-text-white-ter" id="seewhole">
+              SEE THE WHOLE PROGRAM
+            </a>
+          </div>
+        </div>
+      </div>
+
```

Comment:
- [ ] The "Main Program" section appears to deviate from the original design. It should feature a distinctive red line beneath the title, and it should showcase a total of five cards instead of four. Kindly address these issues, paying attention to the appropriate spacing both preceding and following the title, as well as the spacing surrounding the "[SEE THE WHOLE PROGRAM]" text.

Your project - Desktop
<img width="939" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/6712a4f0-b222-4114-a879-702369682db2">

original Design by Cindy
<img width="611" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/6e20e843-44f3-4e5c-af30-94e85a28d87a">

- [ ] When observing the mobile view closely, several issues become apparent:
- The cards seem to lack sufficient padding or margin space on both the left and right sides.
- The alignment of content within each card should be horizontal. This implies that the elements within each card, such as the image, title, and description, should be organized in a row-oriented fashion. The image should be positioned on the left, the title centered, and the description on the right.
- The text reading **_See the Whole program_** should not be visible when viewed on a mobile device but currently, it is visible.
 
Please take the necessary measures to rectify these concerns and enhance the overall mobile viewing experience.

Your work - Mobile
<img width="264" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/b1d77790-6855-4ba4-94c3-06957856578c">

Original design - mobile
![image](https://github.com/Drew246/capstone-project-1/assets/84156714/6dce9749-82fb-4c89-82b8-9ef1a9de60dc)